src/containers/DataGUI.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
:smiley: You have write access to this branch, feel free to push a commit :innocent: